src/allmydata/test/test_crawler.py

```diff
@@ -1,4 +1,18 @@
+"""
+Tests for allmydata.storage.crawler.
+
+Ported to Python 3.
+"""
+
 from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from future.utils import PY2, PY3
+if PY2:
+    # Don't use future bytes, since it breaks tests.
```

Comment:
Does it break them in a way that requires further porting effort?  Or does it break them because future bytes isn't exactly the same as Python 2 bytes - so that once we're on Python 3 and there's only one bytes type everything is okay?

Reply:
The tests pass on Python 3, so Python 3 bytes on Python 3 work just fine. The issue is Python 3 backported bytes on Python 2, so it's not worth investigating.